.circleci/main.yml

```diff
@@ -1452,6 +1452,12 @@ jobs:
       - install-helm
       - install-kubectl
       - start-minikube
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Install pachctl
+          command: |
+            sudo mv ./dist-pach/pachctl/pachctl_linux_amd64_v1/pachctl /usr/local/bin/pachctl
```

Comment:
sorry maybe im reading this wrong. wouldn't this need to have this job run after pachctl has built. to do so would need a `requires` on the job

Reply:
Yeah the requires already is in place, I just forgot to actually "install" pachctl for this job. I figured since now core-team members are more likely to touch these jobs it would be useful to have pachctl actually present if they want to ssh in.